src/list.scss

```diff
@@ -101,6 +101,107 @@ $block: #{$fd-namespace}-list;
         background: var(--sapList_Background);
       }
     }
+
+    &--action {
+      display: -webkit-box;
+      display: flex;
+      size: 2.5rem;
+      padding: 0 0.25rem;
+      -webkit-box-pack: center;
+      justify-content: center;
+      cursor: pointer;
+      border: var(--sapPageFooter_BorderColor);
+      background: var(--sapPageFooter_Background);
+      height: 3rem;
+
+      @include fd-ellipsis();
+
+      &--compact {
+        size: 1.625rem;
+        font-size: var(--sapFontSize);
+      }
+
+      .#{$block}__title {
+        -webkit-box-flex: 1;
+        flex: auto;
+        width: auto;
+        min-width: 3rem;
+        display: block;
+        text-align: center;
+        font-weight: normal;
+        color: var(--sapButton_TextColor);
+        font-size: var(--sapFontLargeSize);
+        text-shadow: none;
```

Comment:
![Screen Shot 2020-04-16 at 5 37 09 PM](https://user-images.githubusercontent.com/39598672/79509156-f531d980-8008-11ea-9fe2-a04fcd97d135.png)

The specs say that "Font size and font family should inherited from the list Item base." You should only change the color and the text-align. The font-weight is coming from reset

Reply:
Changes done.